### PR TITLE
Column1

### PR DESCRIPTION
**Pull request by bhargaviks**
Since it's a holiday, I didn't commit very often. But this is my work for now. I have to work on indentation. But otherwise, I hope it's good enough otherwise. Have to make a lot more changes tomorrow. Just felt I need to create a pull request now.
Thank you for any feedback!

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -34,11 +34,90 @@ <h1 class="title"> Bhargavi Kumar Sundaresan <i class="fas fa-hand-spock"></i></
...
         </div>
       </section>
+
+      <section class = "section">
+        <div class="container">
+          <div class = "columns"> 
+            
+            <div class= "column">
+              <!-- This is column 1 content --> 
+			        <article class="message is-info">
```
Comment (pengxianzhe) [inline portfolio/src/main/webapp/index.html]: Did you copy this part of code from somewhere? The indentation is broken in index.html, but other files seem to be fine.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -11,34 +11,301 @@
...
+			  </div>
+		  </div>
+	  </section>
+    
+    <section class = "section">
+      <div class="container">
+        <div class = "columns"> 
+            
+          <div class= "column">
+            <!-- This is column 1 content -->
```
Comment (pengxianzhe) [inline portfolio/src/main/webapp/index.html:47]: IMO the comments can be more helpful. You can add what each column does in the comment.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -11,34 +11,301 @@
...
-    <section class="section">
-      <div class="container ">
-        <div class="notification is-info is-light">
-          <h1 class="title"> Bhargavi Kumar Sundaresan <i class="fas fa-hand-spock"></i></h1>
-          <p class="subtitle is-italic"> My portfolio!</p><br>
-          </div>
-        </div>
-
-        <div><br></div>
+  <body class="has-background-grey-lighter">
```
Comment (pengxianzhe) [inline portfolio/src/main/webapp/index.html:14]: Did you copy this code from somewhere? The indentation in index.html is messy, but it is fine for other files.

Comment (bhargaviks) [reply portfolio/src/main/webapp/index.html:14]: Bulma has some examples. I just use them to fit my needs and continue writing inside it. I am working on indents and comments now. Will send it to you as soon as I am done! :)

**Pull request merged**